content/flux/v0.x/data-types/basic/string.md

```diff
@@ -0,0 +1,193 @@
+| 2021-01-01T03:00:00Z |                                             3 |
+| 2021-01-01T04:00:00Z |                                             4 |
+{{% /flex-content %}}
+
+{{% flex-content %}}
+##### The example above returns:
+| \_time               | level _<span style="opacity:.5">(string)</span>_ |
+| :------------------- | -----------------------------------------------: |
+| 2021-01-01T00:00:00Z |                                                1 |
+| 2021-01-01T02:00:00Z |                                                2 |
+| 2021-01-01T03:00:00Z |                                                3 |
+| 2021-01-01T04:00:00Z |                                                4 |
+{{% /flex-content %}}
+{{< /flex >}}
+
+## Operate on strings
+Use functions in the [Flux strings package](/flux/v0.x/stdlib/strings/) to perform
+operations including:
+
+- [Compare two strings](/flux/v0.x/stdlib/strings/compare/)
+- [Replace substrings in a string](/flux/v0.x/stdlib/strings/replace/)
+- [Return the length of a string](/flux/v0.x/stdlib/strings/strlen/)
+- [Split a string into an array](/flux/v0.x/stdlib/strings/split/)
+- [Join an array into a string](/flux/v0.x/stdlib/strings/joinstr/)
+- [and more](/flux/v0.x/stdlib/strings/)
+
+## Interpolate strings
+To interpolate strings in a Flux string, enclose embedded [expressions](/flux/v0.x/spec/expressions/)
+in a dollar sign and curly braces `${}`.
+Flux replaces the placeholder with the result of the embedded expression and
+returns a string literal.
+
+```js
+name = "John"
+"My name is ${name}."
+// My name is John.
+
+d = 1m
+"the answer is ${d}"
+// the answer is 1m
+
+t0 = 2016-06-13T17:43:50Z
+"the answer is ${t0}"
+// the answer is 2016-06-13T17:43:50.000000000Z
+
+p = {name: "John", age: 42}
+"My name is ${p.name} and I'm ${p.age} years old."
+// My name is John and I'm 42 years old.
+```
+
+_String interpolation expressions must satisfy the
+[Stringable constraint](/flux/v0.x/spec/types/#stringable-constraint)._
+
+## Concatenate strings
+To concatenate Flux strings, use the `+` operator between string values or
+[expressions](/flux/v0.x/spec/expressions/) that resolve to strings.
+Flux resolves expressions and returns a single concatenated string.
+
+{{% note %}}
+Concatenated expressions must resolve to strings.
+{{% /note %}}
+
+```js
+name = "John"
+"My name is " + name + "."
+// My name is John.
+
+d = 1m
+"the answer is " + string(v: d)
+// the answer is 1m
+
+t0 = 2016-06-13T17:43:50Z
+"the answer is " + string(v: t0)
+// the answer is 2016-06-13T17:43:50.000000000Z
+
+p = {name: "John", age: 42}
+"My name is " + p.name + " and I'm " + string(v: p.age) + " years old."
+// My name is John and I'm 42 years old.
+```
+
+{{% note %}}
+#### String interpolation vs concatenation
+Flux supports both [string interpolation](#interpolate-strings) and
+[string concatenation](#concatenate-strings).
+String interpolation is a more concise method for achieving the same result.
+{{% /note %}}
```

Comment:
Not sure these lines are necessary. Sometimes using an operator `+` makes certain code more readable, no?

Reply:
Yeah, you're right. I'll remove this note.